homeassistant/components/nws_alerts/__init__.py

```diff
@@ -0,0 +1,21 @@
+"""The NWS Alerts integration."""
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+PLATFORMS: list[str] = ["sensor"]
```

Comment:
Please import and use the Platform enum.
```suggestion
PLATFORMS: list[str] = [Platform.SENSOR]
```